packages/wonder-blocks-core/src/components/view.js

```diff
@@ -45,16 +49,23 @@ const StyledSection = addStyle<"section">("section", styles.default);
 
 /**
  * View is a building block for constructing other components. `View` roughly
- * maps to `div` and `Text` roughly maps to `span`. You can override which tag
- * is used to render the component (for semantic purposes) by specifying the
- * `tag` prop.
+ * maps to `div`. You can override which tag is used to render the component
+ * (for semantic purposes) by specifying the `tag` prop.
  *
  * These components can take styles (via the `style` prop) in a variety of
  * manners:
  *
  * - An inline style object
  * - An `aphrodite` StyleSheet style
  * - An array combining the above
```

Comment:
One thing that's missing that should probably be add is a description of the default styling for `View`s, in particular:
- use flexbox by default
- flex-direction is set to `column`
- they each get their own stacking context

Additionally, you may want to include what the inspiration and rationale were for these choices.